graphql_compiler/tests/test_input_data.py

```diff
@@ -29,7 +30,7 @@ def immediate_output():  # noqa: D103
     expected_output_metadata = {
         "animal_name": OutputMetadata(type=GraphQLString, optional=False),
     }
-    expected_input_metadata = {}
+    expected_input_metadata: Dict[str, GraphQLSchemaFieldType] = {}
```

Comment:
Not sure if this is adding to readability or anything. Was it necessary for type checking to work?